fuel-coroutines/src/main/kotlin/com/github/kittinunf/fuel/coroutines/Coroutines.kt

```diff
@@ -9,7 +9,7 @@ private suspend fun <T : Any, U : Deserializable<T>> Request.await(
         deserializable: U
 ): Triple<Request, Response, Result<T, FuelError>> =
         suspendCancellableCoroutine { continuation ->
-            continuation.invokeOnCompletion {
+            continuation.invokeOnCancellation {
                 if (continuation.isCancelled) {
```

Comment:
Should the continuation status be checked in the `if` statement?

From the [release notes](https://github.com/Kotlin/kotlinx.coroutines/blob/master/CHANGES.md#version-0230):
> There is no need to check for cont.isCancelled in a typical handler code anymore (since handler is invoked only when continuation is cancelled).